src/Nethermind/Nethermind.Consensus.AuRa/Transactions/GeneratedTxSourceSealer.cs

```diff
@@ -22,45 +22,40 @@
 using Nethermind.Core;
 using Nethermind.Crypto;
 using Nethermind.State;
+using Nethermind.TxPool;
 using Nethermind.Wallet;
 
 namespace Nethermind.Consensus.AuRa.Transactions
 {
-    public class GeneratedTxSourceApprover : ITxSource
+    public class GeneratedTxSourceSealer : ITxSource
     {
         private readonly ITxSource _innerSource;
-        private readonly IBasicWallet _wallet;
-        private readonly ITimestamper _timestamper;
+        private readonly ITxSealer _txSealer;
         private readonly IStateReader _stateReader;
-        private readonly int _chainId;
+        private readonly ISigner _signer;
 
-        public GeneratedTxSourceApprover(ITxSource innerSource, IBasicWallet wallet, ITimestamper timestamper, IStateReader stateReader, int chainId)
+        public GeneratedTxSourceSealer(ITxSource innerSource, ITxSealer txSealer, IStateReader stateReader, ISigner signer)
```

Comment:
we have two terms:
BlockSealer
TxSigner
you sign transactions, you seal blocks, you can sign some block data as part of the block sealing
ITxSealer <- this does not sound right